api/import.go

```diff
@@ -19,10 +20,24 @@ import (
 //
 
 func ImportPost(post *model.Post) {
-	post.Hashtags, _ = model.ParseHashtags(post.Message)
+	for utf8.RuneCountInString(post.Message) > 0 {
+		var remainder string
+		if utf8.RuneCountInString(post.Message) > model.POST_MESSAGE_MAX_RUNES {
```

Comment:
@grundleborg I'm seeing that you are calling `utf8.RuneCountInString(post.Message)` a couple of times, maybe it will be best if you assign that to a variable instead. 

to quote @jwilander here:
"Might not make much perf difference but it will be better for maintainability"